dap-launch.el

```diff
@@ -28,8 +28,8 @@
 (require 'f)
 (require 'rx)
 
-(defun dap-launch-remove-comments ()
-  "Remove all C-style comments in the current buffer.
+(defun dap-launch-sanitize-json ()
+  "Remove all C-style comments and trailing commas in the current buffer.
```

Comment:
Shouldn't you need to go to `(point-min)` before doing `re-search-forward`?
Also wrap this one inside `save-excursion` too?

Reply:
From the docstring:
Replacement starts at point, and strings before it are ignored,
so you may want to move point to `point-min' with `goto-char'
first. This function moves `point'. Both // and /**/ comments are
supported.

That is as intended.